device/mellanox/x86_64-nvidia_sn5600-r0/platform.json

```diff
@@ -1245,13 +1245,6 @@
                 "2x400G[200G,100G,50G,40G,25G,10G]": ["etp64a", "etp64b"],
                 "4x200G[100G,50G,25G,10G]": ["etp64a", "etp64b", "etp64c", "etp64d"]
             }
-        },
-        "Ethernet512": {
```

Comment:
what's the motivation to move this (index, lanes, breakout_modes) to hwsku.json?

Reply:
It was an attempt to fix an issue. It is reverted back.